### PR TITLE
build(dev-infra): rename example app to test-app-v9

### PR DESCRIPTION
**Pull request by ahnpnl**
## Summary

- Move `example` to `e2e` and rename to `test-app-v9`

- Make CI run tests for both apps

## Test plan

Green CI and CI runs e2e tests with both test apps

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by ahnpnl** [comment]
Hmm for some reasons I don’t see CI running for this PR

**Comment by wtho** [comment]
[It's run](https://app.circleci.com/pipelines/github/ahnpnl/jest-preset-angular/23/workflows/312887d2-54e9-42d4-bc77-b26f86617d98), but it is not connected to the PR. Should be solved once you add a second or modify the first commit for this PR.

File: .circleci/config.yml [hunk 1]
```diff
@@ -40,7 +40,7 @@ jobs:
           name: Test Library
           command: yarn test:ci
 
-  install-test-example:
+  install-test-example-apps:
```
Comment (wtho) [inline .circleci/config.yml:43]: I think `example` should be removed here

File: .circleci/config.yml [hunk 2]
```diff
@@ -50,16 +50,9 @@ jobs:
       - run: 'sudo yarn global add greenkeeper-lockfile@2'
       - run: 'greenkeeper-lockfile-update'
       - run:
-          name: Install Example Dependencies
+          name: Test Example Apps
```
Comment (wtho) [inline .circleci/config.yml:53]: also `example` should be removed here

File: .circleci/config.yml [hunk 3]
```diff
@@ -69,7 +62,7 @@ workflows:
       - test-library:
           requires:
             - install-dependencies
-      - install-test-example:
+      - install-test-example-apps:
```
Comment (wtho) [inline .circleci/config.yml:65]: again, `example` should be removed here

File: README.md [hunk 4]
```diff
@@ -26,7 +26,7 @@ In `src` directory create `setupJest.ts` file with following contents:
 
 ```ts
 import 'jest-preset-angular';
-import './jestGlobalMocks'; // browser mocks globally available for every test
+import './jest-global-mocks'; // browser mocks globally available for every test
```
Comment (wtho) [inline README.md:29]: Nice, I like the dashed version of the file name!

File: README.md [hunk 5]
```diff
@@ -100,7 +100,7 @@ If you look at your `src/test.ts` (or similar bootstrapping test file) file you'
 
 ## Snapshot testing
 
-**Since version 1.1.0** it's possible to [snapshot test](http://facebook.github.io/jest/docs/snapshot-testing.html#snapshot-testing-with-jest) your Angular components. Please note it's still under active development and may be a subject of change. You can lookup [example](/example/src/app) for details
+**Since version 1.1.0** it's possible to [snapshot test](http://facebook.github.io/jest/docs/snapshot-testing.html#snapshot-testing-with-jest) your Angular components. Please note it's still under active development and may be a subject of change. You can lookup [example](/e2e/example/src/app) for details
```
Comment (wtho) [inline README.md:103]: Again, I would like to move away from the `example` vocabulary

File: e2e/test-app-v9/src/index.html [hunk 6]
```diff
@@ -6,7 +6,7 @@
   <base href="/">
 
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <link rel="icon" type="image/x-icon" href="e2e/example/src/favicon.ico">
```
Comment (wtho) [inline e2e/test-app-v9/src/index.html:9]: This href seems to be wrong

File: README.md [hunk 7]
```diff
@@ -26,7 +26,7 @@ In `src` directory create `setupJest.ts` file with following contents:
 
 ```ts
 import 'jest-preset-angular';
-import './jestGlobalMocks'; // browser mocks globally available for every test
+import './jest-global-mocks'; // browser mocks globally available for every test
 ```
 
 _Note: feel free to copy the [`jestGlobalMocks.ts`](https://github.com/thymikee/jest-preset-angular/blob/master/example/src/jestGlobalMocks.ts) file from the example directory and save it next to the `setupJest.ts` file._
```
Comment (wtho) [inline README.md:32]: Here some references are still wrong

File: e2e/README.md [hunk 8]
```diff
@@ -14,18 +14,12 @@ You run the tests on example apps with `yarn test:e2e` (or `npm run test:e2e)`.
...
 What will happen in order is as following:
 
-1. A bundle (we'll call it `[bundle]`) will be created for `jest-preset-angular` using `npm pack` (`yarn pack` is buggy).
-
-    The `prepare` script will be run, so `clean` and `build` (that is why e2e tests are launched before the others, since it's building as part of the process)
- 
-2. `npm ci` and `npm install --save-dev [bundle]` will be run in each project directory in `e2e` directory.
-3. each test suite in each example project are run according to `jest` config of that project.
+1. Each project will install its dependencies
+2. Each test suite in each example project runs according to `jest` config of that project.
```
Comment (wtho) [inline e2e/README.md:18]: again, better use: *in each **test** project*

Comment (ahnpnl) [reply e2e/README.md]: done

File: README.md (hunk 7 above)
Comment (ahnpnl) [reply README.md]: done

File: e2e/test-app-v9/src/index.html (hunk 6 above)
Comment (ahnpnl) [reply e2e/test-app-v9/src/index.html]: done

File: README.md (hunk 5 above)
Comment (ahnpnl) [reply README.md]: done

File: .circleci/config.yml (hunk 3 above)
Comment (ahnpnl) [reply .circleci/config.yml]: done

File: .circleci/config.yml (hunk 2 above)
Comment (ahnpnl) [reply .circleci/config.yml]: done

File: .circleci/config.yml (hunk 1 above)
Comment (ahnpnl) [reply .circleci/config.yml]: done

**Comment by ahnpnl** [comment]
I removed `example` from other places as well.

**Comment by wtho** [comment]
Follow-up PRs:
* Turn project into Yarn Workspace
* #406

**Pull request merged**